docs/element-templates/README.md

```diff
@@ -212,7 +212,7 @@ The following ways exist to map a custom field to the underlying BPMN 2.0 XML:
 * `property`: Maps to a named property in the BPMN 2.0 XML, i.e. `<bpmn:serviceTask {name}={userInput} />`
 * `camunda:property`: Maps to a `<camunda:property name="{name}" value="{userInput}" />` extension element
 * `camunda:inputParameter`: Maps to `<camunda:inputParameter name="{name}" />`
-* `camunda:outputParameter`: Maps to `<camunda:outputParameter name="{userInput}" />`
+* `camunda:outputParameter`: Maps to `<camunda:outputParameter source="{source}" />`
```

Comment:
`camunda:outputParameter` can't have a property `source`: https://github.com/camunda/camunda-bpmn-moddle/blob/master/resources/camunda.json#L611

What do you mean could be `camunda:in` and `camunda:out` bindings: https://github.com/camunda/camunda-bpmn-moddle/blob/master/resources/camunda.json#L25, which is not the same. In-out bindings are used inside call activities for a different purpose then the generic input/output parameters.

Reply:
Inside the element template binding, `source` is the meant to be the `value` of the outputParameter. For example

```json
{
      "label": "Result Status",
      "description": "The process variable to which to assign the send result to",
      "type": "String",
      "value": "mailSendResult",
      "binding": {
        "type": "camunda:outputParameter",
        "source": "${ resultStatus }"
      }
    }
```

will lead to

```xml
<camunda:outputParameter name="mailSendResult">${ resultStatus }</camunda:outputParameter>
```

Reply:
:rocket:

Reply:
With the upcoming improvements of displaying Input and Output variables in the element template view (cf. https://github.com/camunda/camunda-modeler/issues/1894#issuecomment-688775033) this should be clearer, _what_ is binding to _what_

value -> name of outputParameter,
source -> value of outputParameter

Better documentation would also help a lot.